_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -62,89 +59,121 @@ sudo su postgres
 psql
 ```
 
-```bash
+```sql
 ALTER USER postgres password 'myStrongPassword';
 ```
 
 Quit psql typing `\q`
 
 Exit postgres user typing `exit`
 
-Navigate to main folder inside postgresql/version/
+Navigate to main folder inside `postgresql/version/`
 ```bash
 cd /etc/postgresql/14/main/
 ```
 We need to edit two files, `pg_hba.conf` and `postgresql.conf`
 ```bash
-sudo nano pg_hba.conf
+sudo nano -w pg_hba.conf
 ```
 Scroll down the file and Add `host, all, all, 0.0.0.0/0, md5`, has to be the first line before `local, all, postgres, , peer`
 
 | TYPE | DATABASE | USER | ADDRESS | METHOD |
 | ---- | -------- | ---- | ------- | ------ |
 | host | all | all | 0.0.0.0/0 | md5 |
 | local | all | postgres |  | peer |
+{: .docs_table}
```

Comment:
```suggestion
```

Reply:
Is not styling the tables appropriate? (I was planning to style the tables in all the docs, some docs already had that style, not sure when/who did it)
example of "table" (GH does format the table with borders, but the docs doesn't)
<img width="501" alt="image" src="https://user-images.githubusercontent.com/64530191/183234402-83233503-b6ef-4c40-8f82-4a10cf26aa7a.png">

Reply:
Got it. What is the benefit of a table without borders? To me it seems less readable.